src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void apply(Map<String, Integer> storage, FruitTransaction transaction) {
+        storage.merge(
+                transaction.getFruit(),
+                transaction.getQuantity(),
+                Integer::sum
+        );
+    }
```

Comment:
It seems like `ReturnOperation` class is missing package organization according to the checklist. Consider moving this class into a `strategy` package to group all operation handlers together.